tests/infra_test.go

```diff
@@ -303,9 +301,22 @@ var _ = Describe("Infrastructure", func() {
 			keys := getKeysFromMetrics(metrics)
 			for _, key := range keys {
 				// we don't care about the ordering of the labels
-				Expect(key).To(ContainSubstring(fmt.Sprintf("node=%s", nodeName)))
-				Expect(key).To(ContainSubstring(fmt.Sprintf("namespace=%s", vmi.Namespace)))
-				Expect(key).To(ContainSubstring(fmt.Sprintf("name=%s", vmi.Name)))
+				Expect(key).To(ContainSubstring(fmt.Sprintf("node=\"%s\"", nodeName)))
```

Comment:
maybe you can use ` so no need for escape \"

Reply:
You can simplify this taking advance of ContaineSubstring alredy do fmt.Sprintf for you and gomega composition with SatisfyAll
Expect(key).To(SatisfyAll(ContainSubstring(`node="%s"`, nodeName), ContainSubstring...)

Reply:
ack, on it